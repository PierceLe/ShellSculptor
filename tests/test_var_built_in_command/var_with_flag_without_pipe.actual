>> cd /
>> var -s root "ls"
>> echo ${root}
Applications Library System Users Volumes bin cores dev etc home opt private sbin tmp usr var

