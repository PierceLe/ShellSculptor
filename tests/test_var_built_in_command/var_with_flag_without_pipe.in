cd /
var -s root "ls"
echo ${root}
