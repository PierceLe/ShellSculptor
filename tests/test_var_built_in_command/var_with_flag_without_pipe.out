>> cd /
>> var -s root "ls"
>> echo ${root}
bin boot build course dev etc home lib lib64 mnt opt proc root run sbin services srv sys tmp usr var

